prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -457,33 +440,13 @@ class FHIRRouter(
         )
 
         if (!passes) {
-            val filterToLog = "${
-                if (isDefaultFilter(filterType, filters)) {
-                    "(default filter) "
-                } else {
-                    ""
-                }
+            val filterToLog = "${ ""
             }${failingFilterName ?: "unknown"}"
```

Comment:
I think this should work:
```suggestion
            val filterToLog = failingFilterName ?: "unknown"
```